lib/common/profile.js

```diff
@@ -30,66 +36,20 @@ function profileServiceFactory(FileLoader, Constants, Promise, _, waterline, Log
      * @extends {FileLoader}
      */
     function ProfileService () {
-        this.loader = new FileLoader();
+        dbRenderable.call(this);
+        this.directory = Constants.Profiles.Directory;
+        this.collectionName = 'profiles';
```

Comment:
Part of me is wondering whether we would be better off with something like `waterline.profiles.identify` rather than a magic collection string, but part of me is reluctant to include that as a dependency at the same time.